### PR TITLE
color change and fade in effect added

### PR DESCRIPTION
**Pull request by lokesh541**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by lokesh541** [comment]
hi, added the requested styles hope it helps

File: main.css [hunk 1]
```diff
@@ -84,3 +84,12 @@ input[type="number"] {
 		transform:rotate(10deg)
 	}
 }
+
+#error-message{
+  color: #D8000C;
+  animation: fadeIn 5s;
```
Comment (jemimaabu) [inline main.css]: 5s is too long for a transition. Try using 1s or 500ms instead.

File: main.css [hunk 2]
```diff
@@ -84,3 +84,12 @@ input[type="number"] {
 		transform:rotate(10deg)
 	}
 }
+
+#error-message{
```
Comment (jemimaabu) [inline main.css:88]: There's no opacity rule on the div so the transition might not be smooth 
```suggestion
#error-message{
     opacity: 0;
     animation: fadeIn 1s forwards;
}
```

**Review by jemimaabu — CHANGES_REQUESTED** [review]

File: main.css (hunk 1 above)
Comment (lokesh541) [reply main.css]: commited your suggested changes.

Comment (jemimaabu) [reply main.css]: The declarations are outside the rule-set.
Also, the color is not included in the new rules

File: main.css [hunk 3]
```diff
@@ -84,3 +84,13 @@ input[type="number"] {
...
```
Comment (lokesh541) [inline main.css:88]: moved color inside the declaration block

**Review by jemimaabu — APPROVED** [review]

**Pull request merged**